tests/console/check_system_info.pm

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Check system info like `/etc/issue` or `/etc/os-release`.
+# Maintainer: Alynx Zhou <alynx.zhou@suse.com>
+
+use base "basetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console('root-console');
+
+    script_output("SUSEConnect --list");
```

Comment:
Because the output may be out of one screen and using `script_output` can print them to serial port.

Reply:
You can redirect output in the `assert_script_run` too using ` > /dev/$serialdev`. But my question is more about if we want just print somethingor we want to check the output and have the test for it.

Reply:
Thank you for your advice. Currently we just want to print something, because we don't have source to compare with those output. Those code is providing a way for us to check those things manually while reviewing.

Reply:
Sure, then let's keep that in mind for next steps. Cheers ;)